### PR TITLE
RESOURCE-116 Support azure service fabric mesh service replicas

### PR DESCRIPTION
**Pull request by sathish-progress**
### Description

Please describe what this change achieves. Ensure you have read the [Contributing to InSpec](https://github.com/inspec/inspec-azure/CONTRIBUTING.MD) document before submitting.

### Issues Resolved

List any existing issues this PR resolves, or any Discourse or StackOverflow discussion that's relevant

### Check List

- [X] New functionality includes tests
- [X] All tests pass
- [X] `rake lint` passes
- [X] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>

### CODE REVIEW

**Review by soumyo13 — COMMENTED** [review]
Terraform Files are missing.

**Review by soumyo13 — CHANGES_REQUESTED** [review]
Terraform files are missing.

**Review by soumyo13 — APPROVED** [review]

File: test/integration/verify/controls/azure_service_fabric_mesh_replica.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+control 'test the properties of all Azure Service Fabric Mesh Service Replica' do
```
Comment (soumyo13) [inline test/integration/verify/controls/azure_service_fabric_mesh_replica.rb:1]:
```suggestion
skip_control 'test the properties of all Azure Service Fabric Mesh Service Replica' do
```

File: test/integration/verify/controls/azure_service_fabric_mesh_replicas.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+control 'test the properties of all Azure Service Fabric Mesh Service Replicas' do
```
Comment (soumyo13) [inline test/integration/verify/controls/azure_service_fabric_mesh_replicas.rb:1]:
```suggestion
skip_control 'test the properties of all Azure Service Fabric Mesh Service Replicas' do
```

**Review by IanMadd — APPROVED** [review]

**Pull request merged**